app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/claims/confirmation_of_submission.scala.html

```diff
@@ -33,40 +33,65 @@
 
   @mainTemplate(title = title, userType = request.userType) {
 
-      <div class="govuk-box-highlight">
-        <h1 class="heading-xlarge">@title</h1>
-          <p>
-              <span class="tax-owed-label">
-                 @messages(s"$key.reimbursement-amount")
-              </span>
-                  <span class="heading-xlarge">
-                 @{formatAmountOfMoneyWithPoundSign(justSubmittedClaim.completeClaim.totalClaim.toDouble)}
-              </span>
-          </p>
-          <p id="ref-id">
-              @messages(s"$key.claim-reference")
-              <span class="heading-large">
-              @justSubmittedClaim.submissionResponse.caseNumber
-        </span>
-          </p>
-      </div>
-
-    <p>@Html(messages(s"$key.email-sent"))</p>
-
-    <p><span class="heading-large">What happens next?</span></p>
-
-    <p>@Html(messages(s"$key.what-next.p1"))</p>
-
-    <ul class="list-bullet">
-        <li>@Html(messages(s"$key.what-next.l1"))</li>
-        <li>@Html(messages(s"$key.what-next.l2"))</li>
-        <li>@Html(messages(s"$key.what-next.l3"))</li>
-    </ul>
-
-    <p>@Html(messages(s"$key.what-next.p2"))</p>
-
-    <p><a href="@routes.StartController.start().url">@messages(s"$key.what-next.link.1.start-another-claim")</a></p>
-
-    <p><a href="@routes.FeedbackController.feedback().url">@messages(s"exit-survey.link-text")</a></p>
+
+@*
+* Temp code before switch to new frontend
+*@
+
+<link rel="stylesheet" href='/claim-for-reimbursement-of-import-duties/assets/stylesheets/placeholder-govuk.css' />
+
+
+<!--<main class="govuk-main-wrapper govuk-main-wrapper&#45;&#45;s" id="main-content" role="main">-->
```

Comment:
if you don't need this commented out code then please remove it

Reply:
Done